code/src/components/MovieList.jsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import styled from 'styled-components'
```

Comment:
I like that you used styled components. It makes the code in the components much easier to follow, I think.